graphql_compiler/tests/test_data_tools/integration_data/create_orientdb_integration.sql

```diff
@@ -1,5 +1,5 @@
 # Copyright 2018-present Kensho Technologies, LLC.
-create vertex Animal set name = 'Animal 1', net_worth = Decimal('100'), uuid = 'cfc6e625-8594-0927-468f-f53d864a7a50'
-create vertex Animal set name = 'Animal 2', net_worth = Decimal('200'), uuid = 'cfc6e625-8594-0927-468f-f53d864a7a51'
-create vertex Animal set name = 'Animal 3', net_worth = Decimal('300'), uuid = 'cfc6e625-8594-0927-468f-f53d864a7a52'
-create vertex Animal set name = 'Animal 4', net_worth = Decimal('400'), uuid = 'cfc6e625-8594-0927-468f-f53d864a7a53'
+create vertex Animal set name = 'Animal 1', net_worth = Decimal('100'), uuid = 'cfc6e625-8594-0927-468f-f53d864a7a51'
```

Comment:
This was just to make the test data easier to remember, ie `Animal 1` has UUID ending in 1